lib/internal/Magento/Framework/App/Bootstrap.php

```diff
@@ -269,6 +269,8 @@ public function run(AppInterface $application)
             }
         } catch (\Exception $e) {
             $this->terminate($e);
+        } catch (\Error $e) {
```

Comment:
Why not just change line 270 to `} catch (\Throwable $e) {` to make sure we catch everything?

Reply:
agreed - however to be honest I thought about it but I left it like that for better visibility so that the Exception catching is left intact, as that is more common knowledge